apps/events.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+key: events
+title: Events
+---
+
+Events are triggered when a specific action is done.
+So, if you implement one of these events, your app will be informed about related actions and can react accordingly.
+This can be helpful if your app needs information like the current page the user is on, or the fact that a product was added to the cart.
```

Comment:
This can be helpful if your app requires information such as the current page a user is on, or if a product was added to the cart.